Makefile

```diff
@@ -224,13 +224,6 @@ logs: ## display app logs (follow mode)
 	@$(COMPOSE) logs -f app
 .PHONY: logs
 
-run: ## alias for run-es run-mongo run-clickhouse
-run: \
-	run-es \
-	run-mongo \
-	run-clickhouse
-.PHONY: run
```

Comment:
Shouldn't we make this `run` rule an alias to `run-lrs`? To be consistent across our projects (`make run` stands for a server).